src/cmd.rs

```diff
@@ -1052,8 +1166,8 @@ pub fn mgr_disable_module(cluster_handle: &Rados, module: &str, simulate: bool)
     Ok(())
 }
 
-/// dump metadata for all daemons
-pub fn mgr_metadata(cluster_handle: &Rados) -> RadosResult<MgrMetadata> {
+/// dump metadata for all daemons.  Note this only works for Luminous+
+pub fn mgr_metadata(cluster_handle: &Rados) -> RadosResult<Vec<MgrMetadata>> {
```

Comment:
if this is luminous and newer i think the macro should be added here:
`min_version!(Luminous, self);`

Reply:
Isn't that macro for ceph_client though?